db/migrate/20200514193029_add_lockouts_to_phone_number_authentication.rb

```diff
@@ -0,0 +1,6 @@
+class AddLockoutsToPhoneNumberAuthentication < ActiveRecord::Migration[5.2]
+  def change
+    add_column :phone_number_authentications, :failed_attempts, :integer, default: 0, null: false
```

Comment:
Even though this is for internal use, should we back these constraints up with model validations? Fragmented constraints have bitten us before (eg. this is why our data generation is done through the API, as you pointed out).